cli/src/test/scala/com/ibm/sparktc/sparkbench/OutputTest.scala

```diff
@@ -19,18 +19,13 @@ class OutputTest extends FlatSpec with Matchers with BeforeAndAfterAll with Capt
   }
 
   "Specifying Console output" should "work" in {
-    val (out) = captureOutput(CLIKickoff.main(
-      Array("bin/spark-bench.sh", "workload", "kmeans", "-i", dataStuff.kmeansFile, "-o", "console", "-n", "5", "-k", "2"))
-    )
+    val (out) = captureOutput(CLIKickoff.main(Array(getClass.getResource("/etc/testConfFile3.conf").getPath)))
```

Comment:
Er, it looks like you're still using function `captureOutput` in trait `Capturing` that returns a `(String, String)`.  I'm not sure what's going on here syntactically, but I converted `(out)` to `(out, _)` and it still compiles and tests fine.  Does that make any sense?